stats/stats/src/charts/chart.rs

```diff
@@ -104,22 +189,63 @@ pub trait ChartProperties: Sync + Named {
     }
 }
 
+#[macro_export]
+macro_rules! delegated_properties_with_resolutions {
```

Comment:
nit: I didn't get why its called `delegated_properties_with_resolutions`. maybe `define_and_impl_resolutions`

Reply:
Because the new properties' implementations are delegated to the specified one, but the resolution is overwritten. 

`define_and_impl_resolutions` seems to suggest that we define some resolutions, which is not what happens here (?)